src/components/pages/Home.js

```diff
@@ -3,7 +3,9 @@ import React from 'react';
 function Home() {
   return (
     <div>
-      Home
+      <h1 id="home">Home</h1>
```

Comment:
It's common practice to separate shared components and pages, so you chose a good approach here. But Usually you would have the pages and components directories on the same level.

For example like this

```
- src
-- components
--- Header
---- Header.js
-- pages
--- Home
---- Home.js
```